### PR TITLE
Release v0110 prep

### PR DESCRIPTION
**Pull request by sarangan12**
@veronicagg @vishrutshah @salameer 

1. No changes to the runtimes
2. Regenerated with autorest:
    Bootstrapper :        0.9.13
   NetCore framework :   1.0.3
   AutoRest core :       1.0.1-20170301-2300-nightly
3. Executed autorest with -SkipValidation flag
4. Executed arm:rspec. No errors found. 
5. Updated changelog file 
6. VCR step ignored as there are no changes to runtime
8. CHanged the azure storage dependency version

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## 2017.03.10 - Azure ARM version 0.10.0
```
Comment (vishrutshah) [inline ChangeLog.md]: minor: Date 2017.03.06 - as we'll be releasing on 06

File: ChangeLog.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+
+All issues associated with this milestone can be found using this [filter](https://github.com/Azure/azure-sdk-for-ruby/issues?utf8=%E2%9C%93&q=milestone%3Av0.10.0-arm%20).
+
+* Regened all azure_mgmt_* sdks using AutoRest.1.0.1-20170301-2300-nightly for release of ARM version 0.10.0
+* Updated [azure_mgmt_network](https://rubygems.org/gems/azure_mgmt_network) with (Network Watchers API)(https://github.com/Azure/azure-rest-api-specs/blob/master/arm-network/2016-09-01/swagger/networkWatcher.json)
+* Updated [azure_mgmt_batch](https://rubygems.org/gems/azure_mgmt_batch) to latest API version [2017-01-01](https://github.com/Azure/azure-rest-api-specs/blob/master/arm-batch/2017-01-01/swagger/BatchManagement.json)
+* Bug fixes and updated swagger specs
```
Comment (vishrutshah) [inline ChangeLog.md]: So we don't have any notable breaking changes apart from these 2 changes?

File: ChangeLog.md (hunk 1 above)
Comment (sarangan12) [inline ChangeLog.md]: Done

File: ChangeLog.md (hunk 2 above)
Comment (sarangan12) [inline ChangeLog.md]: Added all the changes

**Pull request merged**